Source/System.swift

```diff
@@ -27,7 +27,7 @@ import UIKit
 internal protocol SystemType: Configurable {
     var fileManager: FileManagerType { get }
     var calendar: CalendarType { get }
-    var adIdManager: ASIdentifierManagerType { get }
```

Comment:
why was this removed?

Reply:
We want `adIdManager` to be only accessed by the `System` class, so I had to move the property into the `System` class because the `SystemType` protocol wouldn't allow for a private property.

Reply:
@danielx0328 The ifa is now optional in requests depending on user & brand configuration (e.g. user disabled in Settings, brand sets to false). Given that, it makes sense to enforce proper usage internally in the library. We don't want to expose access to the adIdManager directly or we might inadvertently attach an ifa without checking the two above conditions. The change here reflects that—as the system now exposes the `advertisingId` directly as an optional and returns a value based on those conditions. All consumers must use the correct property on System and the conditions are guaranteed to considered before returning the value.